tests/Nox.Solution.Tests/YamlFileValidationTests.cs

```diff
@@ -42,6 +42,44 @@ public void When_AzureServiceBus_AzureServiceBusConfig_Is_Required()
         Assert.Throws<NoxYamlValidationException>(() => noxConfigBuilder.Build());
     }
 
+    [Fact]
```

Comment:
Can we also add an additional test where both SharedAccessKey and SharedAccessKeyName are set?